src/navigate-pop/navigate-pop.md

```diff
@@ -0,0 +1,15 @@
+<!-- reference: 
+everything: https://support.system76.com/articles/pop-basics
+cosmic: https://github.com/pop-os/cosmic
+pop shell: https://github.com/pop-os/shell
+ubuntu vs. pop: https://support.system76.com/articles/difference-between-pop-ubuntu/
+-->
+
+# Navigate Pop!\_OS
+
+The Pop!\_OS Desktop environment is based on the GNOME shell and layered with other components called Pop!_Shell and C.O.S.M.I.C. (Computer Operating System Main Interface Components).
+
+- [Manipulate windows using Tiling and Stacking.](tiling-stacking-windows.md)
+- [Use Workspaces to organize running applications.](using-workspaces.md)
+- [Use the Nautilus file browser to locate your files and folders.](navigate-files-folders.md)
+- [Use Desktop menus to locate, launch, and organize installed applications.](launching-applications.md)
```

Comment:
Where'd you get `Desktop` as a proper name from?

From a quick glance at that page, it looks like either the "switch between running apps" stuff needs to be moved off to a different page, or this summary needs to be changed to include that aspect. (I would probably recommend the former.)